parser/src/main/java/me/vertretungsplan/parser/LegionBoardParser.java

```diff
@@ -119,7 +120,12 @@ private JSONArray getTeachers() throws IOException, JSONException {
 	}
 
 	private JSONArray getJSONArray(String url) throws IOException, JSONException {
-		return new JSONArray(httpGet(url, "UTF-8"));
+		try {
+			return new JSONArray(httpGet(url, "UTF-8"));
+		} catch (HttpResponseException e) {
+			e.printStackTrace();
```

Comment:
Teachers and courses can be empty and there can still be changes, as these two fields are not required for a change. I'm currently building `parseLegionBoard()` and there I'm checking whether the changes are empty. But having empty teachers and courses is no problem.

Maybe I should try and catch in getChanges/Courses/... and stop execution there? But this was the easiest solution :P